proposals/simd/BinarySIMD.md

```diff
@@ -166,5 +166,5 @@ The `v8x16.shuffle2_imm` instruction has 16 bytes after `simdop`.
 | `f32x4.convert_u/i32x4`   |    `0xb0`| -                  |
 | `f64x2.convert_s/i64x2`   |    `0xb1`| -                  |
 | `f64x2.convert_u/i64x2`   |    `0xb2`| -                  |
-| `v8x16.shuffle1`          |    `0xc0`| -                  |
-| `v8x16.shuffle2_imm`      |    `0xc1`| s:LaneIdx32[16]    |
\ No newline at end of file
+| `v8x16.swizzle`           |    `0xc0`| -                  |
+| `v8x16.shuffle_imm`       |    `0xc1`| s:LaneIdx32[16]    |
```

Comment:
IIRC, we agreed on swizzle, and shuffle and not shuffle_imm. Not opposed to having an explicit name, but just pointing out in case anyone has strong opinions about this.

Reply:
You're right, the [comment](https://github.com/WebAssembly/simd/issues/78#issuecomment-508282544)  I wrote at the time just uses `shuffle`. I'm not sure why I still had the `_imm` suffix in my head.